lld/test/COFF/Inputs/directives-multiple.asm

```diff
@@ -0,0 +1,43 @@
+; fasm2 directives-multiple.asm
```

Comment:
So the .OBJ below is generated from this .ASM file right? But this file is unused right now by the LLD tests? I'm fine keeping it if you add a little comment at the top explaining the situation, and that the LLVM tooling should be fixed eventually to not have the .OBJ file there.

Reply:
Instead of having a binary `.obj` file, it can be converted into a `.yaml` with `obj2yaml`, which makes it human readable. That should allow constructing any form of object file, with multiple sections of the same type and all.

(One can still keep the original assembly input as a comment within that yaml file though, with instructions on how to generate it.)

Reply:
The test/yaml had me confused initially, but seems ideal to produce rather terse cases.